docroot/sites/default/settings.ddev-overrides.php

```diff
@@ -20,21 +20,12 @@
 assert_options(ASSERT_ACTIVE, TRUE);
 Handle::register();
 
-// Enable local development services.
+// Enable local development services, including the null cache backend.
 $settings['container_yamls'][] = DRUPAL_ROOT . '/sites/development.services.yml';
 
 // Show all error messages, with backtrace information.
 $config['system.logging']['error_level'] = 'verbose';
```

Comment:
I guess this isn't necessary now because the development services are configured in `default.services.local.yml` --> `services.local.yml`. Also, the file `/sites/development.services.yml` is provided by composer via the drupal scaffolding, and occasionally isn't checked in to git, so it can be unstable in some projects...